### PR TITLE
sean completed react stopwatch component

### PR DESCRIPTION
**Pull request by Sean-Hughes-1**
timerq

### CODE REVIEW

File: react-stopwatch-component/src/stopwatch.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  }
+
+  toggleButton() {
+    const button = document.querySelector('.fa-solid');
+    if (button.className === 'fa-solid fa-play') {
+      button.className = 'fa-solid fa-pause';
+    } else {
+      button.className = 'fa-solid fa-play';
+    }
+  }
```
Comment (Zhiwutian) [inline react-stopwatch-component/src/stopwatch.jsx:37]: make sure you update this so you are not interacting with the DOM directly

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one small update!

File: react-stopwatch-component/src/stopwatch.jsx (hunk 1 above)
Comment (Sean-Hughes-1) [reply react-stopwatch-component/src/stopwatch.jsx]: Updated

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**